baseband/mark5b/base.py

```diff
@@ -223,11 +223,15 @@ class Mark5BStreamReader(Mark5BStreamBase, VLBIStreamReaderBase):
     _sample_shape_maker = Mark5BPayload._sample_shape_maker
 
     def __init__(self, fh_raw, sample_rate=None, kday=None, ref_time=None,
-                 nchan=1, bps=2, squeeze=True, subset=(), fill_value=0.):
+                 nchan=None, bps=2, squeeze=True, subset=(), fill_value=0.):
+
+        if nchan is None:
+            raise TypeError("Mark5B stream reader missing required argument "
```

Comment:
I'd word it as:
```
TypeError("Mark 5B stream reader requires nchan.")
```
or
```
TypeError("Mark 5B stream reader requires nchan be explicitly passed.")
```

Reply:
I based myself on what python gives, but I agree your text is better so will change.